.circleci/config.yml

```diff
@@ -65,6 +65,8 @@ jobs:
             composer install
             ./vendor/bin/phpunit
       - run: test -f wptheme.zip
+    environment:
+      AMO_BASE_URL: https://addons-dev.allizom.org
```

Comment:
Maybe add a comment to explain why it is needed here.

Reply:
Rather than trying to explain it in a comment, I included a comment that points back to the discussion in the PR.